### PR TITLE
Work on validating, calculations, etc

### LINKED ISSUE #22: Add a Mech-building Validation page
Add a page that walks players through the process of calculating mech stats and making sure their build works as intended.

- [x] Validating
    - [x] Calculating mech stats
    - [x] What to do if your stats don't work out
    - [x] Tips and tricks for tuning, such as making use of Equipment or Modules to even out stat imbalances

### PR DESCRIPTION
**Pull request by Icosahunter**
Fixes #22

### CODE REVIEW

File: rules/mech-building/validating.md [hunk 1]
```diff
@@ -14,40 +15,13 @@ Double check that:
...
-
-{.table}
-
-| Resource          | Calculation                                                   |
-| ----------------- | ------------------------------------------------------------- |
-| Available CPU     | Sum CPU from Mainframe(s), Compute Module(s)                  |
-| Used CPU          | Sum CPU from Cabin, Power Plant(s), Cooler(s), Other Module(s)|
-| Excess CPU        | Available CPU - Used CPU                                      |
+### Resources
+To make sure you have enough resources check that your available resources is greater than your used resources and that your left over excess resources is enough to use as many of your excess-draw components as you want to in a single turn.
```
Comment (infinitymdm) [inline rules/mech-building/validating.md]: This is a lot for one sentence. Maybe break it into a 2-item list?

File: rules/mech-building/validating.md [hunk 2]
```diff
@@ -14,40 +15,13 @@ Double check that:
 - Large components are in large slots 
 
 ### Power Type
-Before calculating any resources it's a good idea to get the hard constraints out of the way. First up is {% keyword "Power Type" glossary %}. If you are doing a single power plant mech (or a multi-power plant mech but they all produce the same power type), then all your other components must match the power type of your power plant. For mechs with multiple power types see [Multiple Power Plants]({{site_url}}/rules/multiple-power-plants/).
+One of the first things you should check is {% keyword "Power Type" glossary %}. If you are doing a single power plant mech (or a multi-power plant mech but they all produce the same power type), then all your other components must match the power type of your power plant. For mechs with multiple power types see [Multiple Power Plants]({{site_url}}/rules/multiple-power-plants/).
```
Comment (infinitymdm) [inline rules/mech-building/validating.md:18]: Power Type isn't in the glossary at present. You'll either need to add that or remove this keyword. (Pretty sure this is my fault, but still)

File: rules/mech-building/validating.md [hunk 3]
```diff
@@ -14,40 +15,13 @@ Double check that:
...
-| Available CPU     | Sum CPU from Mainframe(s), Compute Module(s)                  |
-| Used CPU          | Sum CPU from Cabin, Power Plant(s), Cooler(s), Other Module(s)|
-| Excess CPU        | Available CPU - Used CPU                                      |
+### Resources
+To make sure you have enough resources check that your available resources is greater than your used resources and that your left over excess resources is enough to use as many of your excess-draw components as you want to in a single turn.
 
-{.table}
\ No newline at end of file
+### Fixing and tuning your build
+If at any point you realize you won't have enough resources to do something, or your stats are otherwise not as desired, you'll of course have to adjust what components you are using. Remember there's more than one way to adjust stats, often times modules and equipment provide alternative ways to adjust resource usage, speed, damage, or other stats and may be used to fine tune a build.
```
Comment (infinitymdm) [inline rules/mech-building/validating.md]: Comma splice in the 2nd sentence here. Start a new sentence at "often" or reword.

File: rules/turn-based-play/weapons.md [hunk 4]
```diff
@@ -4,7 +4,11 @@ title: Weapon Specials
 permalink: rules/weapons/
 tags: rules_page
 ---
-## Weapon Keywords Effects
+
+## Weapons
+Weapons deal damage and special effects during combat. For details on weapon use see [Combat]({{site_url}}/rules/combat/).
```
Comment (infinitymdm) [inline rules/turn-based-play/weapons.md:9]: ~~weapon~~ -> weapons

File: rules/turn-based-play/combat.md [hunk 5]
```diff
@@ -23,6 +23,7 @@ Attacking comes in a few steps like so:
 ### Consequences
 Losing some components will have severe consequences for you.
 If one of your components being damaged/destroyed leaves you with too few resources to power your constant draw components, your mech will no longer function at all.
+A destroyed joint means you can no longer use attachments in that joint, and if the destroyed joint was your in your legs section you can no longer move.
```
Comment (infinitymdm) [inline rules/turn-based-play/combat.md]: Line 26 and 27 may need to be swapped here? As read now, L27 seems to indicate that you can get joints working by disabling components

**Review by infinitymdm — CHANGES_REQUESTED** [review]

File: rules/turn-based-play/combat.md (hunk 5 above)
Comment (Icosahunter) [reply rules/turn-based-play/combat.md]: Fixed

File: rules/mech-building/validating.md (hunk 1 above)
Comment (Icosahunter) [reply rules/mech-building/validating.md]: Fixed

File: rules/mech-building/validating.md (hunk 2 above)
Comment (Icosahunter) [reply rules/mech-building/validating.md:18]: Fixed

File: rules/mech-building/validating.md (hunk 3 above)
Comment (Icosahunter) [reply rules/mech-building/validating.md]: Fixed

**Pull request merged**